### DISCUSSION TITLE
Can I create a static website working on local file system?

### DISCUSSION THREAD

**Discussion by dmenne** [mkdocs/mkdocs#3406] [Q&A]
We have put some work into building a documentation site with Hugo. Development worked well with "Hugo serve", but when we built the site as a static site, we realized that it must be used on a server, otherwise links are hopelessly mixed up. Our client requires that the documentation is strictly local and cannot run a server just for the documentation.

See https://discourse.gohugo.io/t/make-your-site-browsable-from-a-local-file-system/35098/4 for the hack required, and it must be done separately for each theme.

This is a somewhat hidden "feature" of Hugo, and before I look into an alternative, it would be good to know that mkdocs can create strictly local files that can be run by simply clicking on index.html in the file explorer.

**Comment by Andre601** [comment]
Not quite sure if I understand your question right, so correct me if I am wrong, but I understand you want files to exist to browse in your file explorer?
In that case is `mkdocs build` the command to use, as it generate all files in the site directory (by default `site`).

**Comment by oprypin** [comment]
Yes I think most features will work. Search won't work with built-in themes but you can check out this theme's support
https://squidfunk.github.io/mkdocs-material/setup/building-for-offline-usage/

**Reply by dmenne** [reply]
Thanks, that sound like what I can use

**Comment by dmenne** [comment]
We will stick with Hugo, where I found one theme supporting local installation:

https://mcshelby.github.io/hugo-theme-relearn/basics/installation/index.html
